app/css/components/service-list.css

```diff
@@ -4,11 +4,7 @@
 
 .service-list__item {
   display: flex;
-  padding: 0;
   position: relative;
-}
-
-.service-list__item[data-icon] {
```

Comment:
It seems we don't and won't have service list items without icons or I am missing something?